modules/core/src/main/scala/org/tesselation/cli/config.scala

```diff
@@ -0,0 +1,18 @@
+package org.tesselation.cli
+
+import org.tesselation.cli.config.CliMethod.CliMethod
+
+object config {
+
+  object CliMethod extends Enumeration {
+    type CliMethod = Value
+
+    val RunValidator, RunGenesis = Value
+  }
+
+  case class CliConfig(
+    method: CliMethod = null,
+    genesisPath: String = null
```

Comment:
How about `genesisPath: Path`? Imo we should use refined/new/concrete types in both configs

Reply:
good catch :+1: done